TF_Settings_Web/src/Pages/Camera/Masking/MaskingSlider.tsx

```diff
@@ -36,31 +35,22 @@ export const MaskingSlider: React.FC<MaskingSliderProps> = ({ direction, masking
     // ===== UseEffects =====
     useEffect(() => {
         if (!sliderRef.current) return;
-        const top = Number.parseFloat(getComputedStyle(sliderRef.current).top);
-        const left = Number.parseFloat(getComputedStyle(sliderRef.current).left);
+        const size = innerHeight < innerWidth ? innerHeight * 0.45 : innerWidth * 0.96 - innerHeight * 0.1;
```

Comment:
💬 This is the same 'magic value' as in MaskingScreen. Should this be stored somewhere

Reply:
Is this not the same as canvasSize?

Reply:
Yep! Updated to use that instead